### PR TITLE
Add LICENSE

### PR DESCRIPTION
**Pull request by ssmall**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2020 Javier Domene, Hisao Yesaki, and Oscar David Rodríguez Moscosa
```
Comment (ssmall) [inline LICENSE:189]: @oscardavidrm @jaimehisao @Domene99 please confirm that this is how you'd like your names to appear on the license

Comment (Domene99) [reply LICENSE:189]: Fine for me

Comment (oscardavidrm) [reply LICENSE:189]: LGTM!

Comment (jaimehisao) [reply LICENSE:189]: LGTM!

**Pull request merged**